packaging/vagrant/Vagrantfile

```diff
@@ -182,9 +182,11 @@ EOF2
     git clone --recursive https://github.com/hpc/charliecloud.git
 
     cd charliecloud
-    if [[ $CH_VERSION ]]; then
-      git checkout $CH_VERSION
+    if [[ -z $CH_VERSION ]]; then
+      CH_VERSION=$(git tag --sort=-version:refname --format='%(refname:strip=2)' | head -n 1)
     fi
+    git checkout $CH_VERSION
```

Comment:
Would it be useful to print what version of Charliecloud we found somewhere?

Reply:
I think so.